assets/express/src/database/models/user.model.ts

```diff
@@ -0,0 +1,45 @@
+import { UserRepository } from '../interfaces';
+import { User, UserPayload } from '@entities';
+
+// This is just sample implementation at this point, the DB functionality
+// will be added with next tickets
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unused-vars
+export default function userOperations(dbConnection: any): UserRepository {
```

Comment:
This is `user.model.ts` with function name `userOperations` that returns `UserRepository`?